enterprise/views.py

```diff
@@ -774,6 +782,9 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
             organization_logo = None
             organization_name = None
 
+        if enterprise_course_enrollment and not enterprise_course_enrollment.consent_granted:
+            add_consent_declined_message(request, enterprise_customer, course_details)
```

Comment:
@douglashall we can use the util method [is_consent_required_for_user(enterprise_customer_user, course_id=None)](https://github.com/edx/edx-enterprise/blob/master/enterprise/utils.py#L389) here.